conda_build/build.py

```diff
@@ -1802,7 +1804,14 @@ def post_process_files(m: MetaData, initial_prefix_files):
     return new_files
 
 
-def bundle_conda(output, metadata: MetaData, env, stats, **kw):
+def bundle_conda(
+    output,
+    metadata: MetaData,
+    env,
+    stats,
+    new_prefix_files: set[str],
```

Comment:
Since this is adding a new positional argument to a (let's assume) public signature, what about making it a `None` kwarg. This way downstream users won't run into number of arguments errors:

```suggestion
    new_prefix_files: set[str] = None,
```